setup.py

```diff
@@ -1,28 +1,81 @@
-from setuptools import find_packages
-from setuptools import setup
+import logging
+import os
 
-__version__ = '0.1.0'
+from setuptools import find_packages, setup
+from setuptools.command.develop import develop
+from setuptools.command.egg_info import egg_info
+from setuptools.command.install import install
+from sqlalchemy.orm import sessionmaker
```

Comment:
You can't import sqlalchemy in the `setup.py` as it's not yet installed.

If I execute in a clean virtualenv `pip3 install git+git://github.com/lytex/pydo` I get the following error:

```
$: pip3 install git+git://github.com/lytex/pydo                [test](master)
Collecting git+git://github.com/lytex/pydo
  Cloning git://github.com/lytex/pydo to /tmp/pip-req-build-ab8fn25w
  Running command git clone -q git://github.com/lytex/pydo /tmp/pip-req-build-ab8fn25w
    ERROR: Command errored out with exit status 1:
     command: /tmp/tmp.XN2SZgTeVz/test/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-ab8fn25w/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-ab8fn25w/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-req-build-ab8fn25w/pip-egg-info
         cwd: /tmp/pip-req-build-ab8fn25w/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-ab8fn25w/setup.py", line 8, in <module>
        from sqlalchemy.orm import sessionmaker
    ModuleNotFoundError: No module named 'sqlalchemy'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Reply:
Anyway, it seems that you don't use the sessionmaker anywhere so you could remove the line

Reply:
That won't solve the issue since pydo can't run if its dependencies are not installed. Removing that line, the next dependency that fails is argcomplete

```
Collecting git+git://github.com/lytex/pydo
  Cloning git://github.com/lytex/pydo to /tmp/pip-req-build-q2zc7quf
  Running command git clone -q git://github.com/lytex/pydo /tmp/pip-req-build-q2zc7quf
    ERROR: Command errored out with exit status 1:
     command: /home/julian/.virtualenvs/test_pydo/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-q2zc7quf/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-q2zc7quf/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-req-build-q2zc7quf/pip-egg-info
         cwd: /tmp/pip-req-build-q2zc7quf/
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-q2zc7quf/setup.py", line 9, in <module>
        import pydo
      File "/tmp/pip-req-build-q2zc7quf/pydo/__init__.py", line 19, in <module>
        from pydo.cli import load_logger, load_parser
      File "/tmp/pip-req-build-q2zc7quf/pydo/cli.py", line 11, in <module>
        import argcomplete
    ModuleNotFoundError: No module named 'argcomplete'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
I'm going to follow this thread:
https://stackoverflow.com/questions/21915469/python-setuptools-install-requires-is-ignored-when-overriding-cmdclass

Reply:
Solved in [78ad1b6](https://github.com/lyz-code/pydo/pull/60/commits/78ad1b67891b0bee64cc32612ff8bef7ec73894b)